lib/pf/Authentication/Source/ADSource.pm

```diff
@@ -24,6 +24,7 @@ sub available_attributes {
      { value => "sAMAccountName", type => $Conditions::SUBSTRING },
      { value => "sAMAccountType", type => $Conditions::SUBSTRING },
      { value => "userAccountControl", type => $Conditions::SUBSTRING },
+     { value => "memberOf:1.2.840.113556.1.4.1941:", type => $Conditions::SUBSTRING },
```

Comment:
Too much magic number makes my brain hurt. 
Can we put that into a named constant at least?